src/core/lib/security/credentials/google_default/google_default_credentials.cc

```diff
@@ -365,6 +363,8 @@ void grpc_flush_cached_google_default_credentials(void) {
   gpr_mu_unlock(&g_state_mu);
 }
 
+bool is_on_gce(void) { return g_gce_tenancy_checker(); }
```

Comment:
Suggest calling this `running_on_gce()`.

Reply:
Changed.